service/routers/decks.js

```diff
@@ -0,0 +1,184 @@
+import { Router } from 'express'
+import { body } from 'express-validator'
+import { User } from '../models/User.js'
+
+const decksRouter = Router()
+
+const getDecks = async (req, res) => {
+  if(req.user.role == "Admin" || req.user.role == "SuperUser" || req.user.userId == req.params.id) {
```

Comment:
What will this accomplish?

Reply:
This part of the code had the mistake of using req.params.id instead of using req.params.userId. I am so sorry for the mistake professor.